.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,8 @@
 <!-- 
 Describe the changes you have made here: what, why, ... 
-Link issues that are fixed, e.g. "Fixes #333".
-If you fixed a koppor issue, link it, e.g. "Fixes https://github.com/koppor/jabref/issues/47".
+Link issues that are closed, e.g. "Closes #333".
+If you resolved a koppor issue, link it, e.g. "Closes https://github.com/koppor/jabref/issues/47".
+You need to stick with "Closes" as verb, because GitHub recognizes this word (and not other words).
 The title of the PR must not reference an issue, because GitHub does not support autolinking there.
```

Comment:
```suggestion
Link the issues that will be closed, e.g., "Closes [#333](https://github.com/JabRef/jabref/issues/333)".	 
If your PR closes a koppor issue, link it using its URL, e.g., "Closes https://github.com/koppor/jabref/issues/47".
"Closes" is a keyword GitHub uses to link PRs with issues; do not change it.
Don't reference an issue in the PR title because GitHub does not support auto-linking there.
```

Reply:
Take or throw the suggestions, I just got curious about what was in the PR. I am sure something like this could have tripped me up when I got started, i.e., fixes vs resolves vs closes.

Pretty sure I had to ask about it at some point 🤔